src/box.tsx

```diff
@@ -14,6 +15,36 @@ const Box: BoxComponent = ({ is = 'div', innerRef, children, ...props }) => {
     parsedProps.ref = innerRef
   }
 
+  /**
+   * If the user has enabled safe hrefs we want to make sure that the url passed
+   * is safe and that the other attributes that make the link safe are added to the
+   * element
+   */
+  const safeHrefEnabled = !allowUnsafeHref && getUseSafeHref() && is === 'a' && parsedProps.href
+  if (safeHrefEnabled) {
```

Comment:
I'd recommend moving this whole block to a separate named function (was what I was trying to suggest earlier but think it got lost in translation).

Reply:
Something like 
```tsx
if (safeHrefEnabled) {
  const { href, rel } = extractAnchorProps({ allowUnsafeHref, ...parsedProps })
  parsedProps.href = href
  parsedProps.rel = rel
}
```

Reply:
oh lol yes, completely misunderstood what you meant. That makes sense and makes this a little cleaner.

Reply:
Done-zo